proposals/solid-oidc/primer.bs

```diff
@@ -0,0 +1,745 @@
+
+#### 9. Retrieves OP configuration
+
+The RS uses the `iss` claim to get the issuer's configuration. The url is the issuer claim with `/.well-known/openid-configuration` appended to the end.
+
+Request
+```
+GET https://secureauth.example/.well-known/openid-configuration
+```
+
+Response
+```json
+{  
+   "issuer":"https://secureauth.example",
+   "authorization_endpoint":"https://secureauth.example/authorize",
+   "token_endpoint":"https://secureauth.example/token",
+   "userinfo_endpoint":"https://secureauth.example/userinfo",
+   "jwks_uri":"https://secureauth.example/jwks",
+   "registration_endpoint":"https://secureauth.example/register",
+   "response_types_supported":[  
+      "code",
+      "code token",
+      "code id_token",
+      "id_token",
+      "id_token token",
+      "code id_token token",
+      "none"
+   ],
+   "response_modes_supported":[  
+      "query",
+      "fragment"
+   ],
+   "grant_types_supported":[  
+      "authorization_code",
+      "implicit",
+      "refresh_token",
+      "client_credentials"
+   ],
+   "subject_types_supported":[  
+      "public"
+   ],
+   "id_token_signing_alg_values_supported":[  
+      "RS256",
+      "RS384",
+      "RS512",
+      "none"
+   ],
+   "token_endpoint_auth_methods_supported":[  
+      "client_secret_basic"
+   ],
+   "token_endpoint_auth_signing_alg_values_supported":[  
+      "RS256"
+   ],
+   "display_values_supported":[  
+
+   ],
+   "claim_types_supported":[  
+      "normal"
+   ],
+   "claims_supported":[  
+
+   ],
+   "claims_parameter_supported":false,
+   "request_parameter_supported":true,
+   "request_uri_parameter_supported":false,
+   "require_request_uri_registration":false,
+   "check_session_iframe":"https://secureauth.example/session",
+   "end_session_endpoint":"https://secureauth.example/logout"
+}
+```
+
+#### 10. Requests JWKS
+
+Using the `jwks_uri` field in the openid-configuration, the RS makes a request to retreive the OS's public keys.
+
+Request
+```
+GET https://secureauth.example/jwks
+```
+
+Response (application/json)
+```json
+{
+  keys: [
+    {
+      alg: "EC"
+      crv: "P-256"
+      d: "r9vj-KBwanlREOB62F-CXNR5Q_Q-WKkm_9amJDcDcDM"
+      kid: "Xu68Q0ZfwDiRfWOb2UE8N77GoEQQ7q58_3gl1wsKENs"
+      kty: "EC"
+      use: "sig"
+      x: "qBbrIQNVTIm7M88iJFVB3e1GqsbFbYYfFiibkd48_Ac"
+      y: "SCl_hDR_6SEuJhKFOiVo8-zLqNglZ56jiJw2_PNE9hU"
+    }
+  ]
+}
+```
+
+Notice that the `keys` field is an array, so an OP could have multiple public keys.
+
+#### 11. Checks access token signature validity
+
+The RS checks to see if the access token was signed by one of the keys in the key set. If it was not, the RS must reject the request with a 403.
+
+#### 12. Performs Authorization
+
+Now that the RS has performed all of its checks, we can trust that the user in the `sub` claim (https://alice.coolpod.example/profile/card#me) is truely the user that made the request. Using that information, the RS performs authorization.
```

Comment:
```suggestion
Now that the RS has performed all of its checks, we can trust that the agent in the `webid` claim (https://alice.coolpod.example/profile/card#me) is the same agent that made the request. Using that information, the RS performs authorization.
```

(using "truly" might be going a bit far)

Reply:
Or even better: "...is the same agent on whose behalf the request was made"
OIDC is, at its core, a delegation protocol, so the agent whose WebID this is may not be making the request at all